File: simex_app/app/src/main/java/com/example/simex_app/data/models/Usuari.kt
package com.example.simex_app.data.models

import com.google.gson.annotations.SerializedName

data class Usuari(
    @SerializedName("id", alternate = ["Id"]) val id: Int,
    @SerializedName("nom", alternate = ["Nom"]) val nom: String,
    @SerializedName("cognoms", alternate = ["Cognoms"]) val cognoms: String,
    @SerializedName("correu", alternate = ["Correu"]) val correu: String,
    @SerializedName("rolId", alternate = ["RolId"]) val rolId: Int,
    @SerializedName("active", alternate = ["Active"]) val active: Int,
    @SerializedName("dniFotoFrontal", alternate = ["DniFotoFrontal"]) val dniFotoFrontal: String? = null,
    @SerializedName("dniFotoTrasera", alternate = ["DniFotoTrasera"]) val dniFotoTrasera: String? = null
)

File: smiex_api/smiex_api/Models/Usuari.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace smiex_api.Models;

public partial class Usuari
{
    public int Id { get; set; }

    public string Correu { get; set; } = null!;

    public string Contrasenya { get; set; } = null!;

    public string Nom { get; set; } = null!;

    public string Cognoms { get; set; } = null!;

    public int RolId { get; set; }

    public int Active { get; set; }

    public virtual ICollection<Notificacione> NotificacioneIdUserrecieveNavigations { get; set; } = new List<Notificacione>();

    public virtual ICollection<Notificacione> NotificacioneIdUsersendNavigations { get; set; } = new List<Notificacione>();

    public virtual ICollection<Oferte> Ofertes { get; set; } = new List<Oferte>();

    public virtual Rol Rol { get; set; } = null!;

    [Column("dni_foto_frontal")]
    public byte[]? DniFotoFrontal { get; set; }

    [Column("dni_foto_trasera")]
    public byte[]? DniFotoTrasera { get; set; }
}
